test/unit/tracks/text-track-settings.test.js

```diff
@@ -9,6 +9,16 @@ const tracks = [{
   label: 'test'
 }];
 
+const defaultSettings = {
```

Comment:
I'm not a fan of having these default values, I would prefer if vttjs just did whatever it wanted by default unless a user selects a value.
Maybe the `---` values should just change to be called `default`? or something?